pages/glossary/character.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Character
+key: character
```

Comment:
`character` seems like a generic definition that we can refer to elsewhere? Perhaps - https://developer.mozilla.org/en-US/docs/Glossary/Character

I see you drawing some distinction away from `grapheme`, then may be this should not be called `character`?

Reply:
The issue is that computers don't really deal in graphemes. A language such as JavaScript, for example, represents strings of "characters" as sequences of UTF-16 code points, while other languages, such as C and Go, represent strings of "characters" as sequences of arbitrary bytes. The term "character" therefore has lots of different meanings in different contexts.